contracts/extensions/sequential-governance/CoreGovernance.sol

```diff
@@ -165,10 +161,11 @@ abstract contract CoreGovernance is SignatureConsumer, VoteStatusConsumer, Chain
     uint256[] memory _values,
     bytes[] memory _calldatas,
     uint256[] memory _gasAmounts,
-    address _roninTrustedOrganizationContract,
-    address _gatewayContract,
-    address _creator
-  ) internal virtual returns (uint256 _round) {
+    // addressPack[0] _roninTrustedOrganizationContract,
+    // addressPack[1] _gatewayContract,
+    // addressPack[2] _creator,
+    address[] memory addressPack
+  ) internal virtual {
     GlobalProposal.GlobalProposalDetail memory _globalProposal = GlobalProposal.GlobalProposalDetail(
       round[0] + 1,
```

Comment:
Should this be handled the same as the method `_proposeProposal`? I guess the issue is still while `round[0] + 1`.